_git/git_commands.py

```diff
@@ -404,24 +388,26 @@ def extra_commands(GitCommandRuleBuilder):
             'apply', 'branch', 'clear', 'create', 'drop', 'list', 'pop',
             'push', 'save', 'save', 'show', 'store',
         ])
+        .smart_options(['0', '1', '2', '3', '4', '5'])
```

Comment:
Most numbers on  usages of git stash are 0, 1, and  sometimes 2. I didn't think it was worth adding the extra numbers, and polluting the single syllable namespace